_pages/home/spartacus-faq.md

```diff
@@ -3,131 +3,140 @@ title: Spartacus FAQ
+### What about SEO? I heard that single page storefronts result in lower SEO ranking and make it difficult to share pages on social media. Is this an issue with Spartacus?
 
-Not an issue. No storefront would work with SEO. Spartacus is indeed a “SPA” (Single-Page Application), which benefits performance and flexibility while only loading one page. In order to support SEO, Spartacus also supports Server-Side Rendering (SSR), which builds entire pages on the server side before providing to the client. SSR provides web crawlers with access to individual pages for search indexing purposes. SSR also allows users to share a link to a page on things like social media, and Spartacus also allows configuration of social media meta tags. Usage of SSR has the added benefit of greatly speeding up first time-to-view. 
+Not an issue. No storefront would work with SEO. Spartacus is indeed a “SPA” (Single-Page Application), which benefits performance and flexibility while only loading one page. In order to support SEO, Spartacus also supports Server-Side Rendering (SSR), which builds entire pages on the server side before providing to the client. SSR provides web crawlers with access to individual pages for search indexing purposes. SSR also allows users to share a link to a page on things like social media, and Spartacus also allows configuration of social media meta tags. Usage of SSR has the added benefit of greatly speeding up first time-to-view.
 
 Your server must also support SSR functionality. Support for SSR will be added to SAP Commerce Cloud hosting services Q3 2019. 
 
-### Does Spartacus scale? How? 
+### Does Spartacus scale? How?
 
-Spartacus storefronts are JavaScript web applications that communicate to the backend through REST APIs. This means that Spartacus storefronts (or any JavaScript-based storefronts) are decoupled from the backend SAP Commerce Cloud instance; so its nodes can be scaled separately. Other scaling considerations such as a Content Delivery Network (CDN) can also be leveraged to help decrease load. Additionally, with PWA cache-first networking capabilities, you will have the option to cache resources locally on the user’s device. Finally, lazy loading can be used to determine if/when to load portions of your components.   
+Spartacus storefronts are JavaScript web applications that communicate to the backend through REST APIs. This means that Spartacus storefronts (or any JavaScript-based storefronts) are decoupled from the backend SAP Commerce Cloud instance; so its nodes can be scaled separately. Other scaling considerations such as a Content Delivery Network (CDN) can also be leveraged to help decrease load. Additionally, with PWA cache-first networking capabilities, you will have the option to cache resources locally on the user’s device. Finally, lazy loading can be used to determine if/when to load portions of your components.
```

Comment:
I'd drop the line about lazy loading since this is not yet in place.

Reply:
Removed.